source/Coop.Core/Client/CoopClient.cs

```diff
@@ -46,6 +46,7 @@ public class CoopClient : CoopNetworkBase, ICoopClient
         // TODO add configuration
         netManager = new NetManager(this);
 
+        netManager.DisconnectTimeout = 300 * 1000;
 #if DEBUG
         // Increase disconnect timeout to prevent disconnect during debugging
         netManager.DisconnectTimeout = 300 * 1000;
```

Comment:
Why don't we use the ConnectionTimeout field from NetworkConfiguration to set disconnect timeouts?

i.e.
```cs
netManager.DisconnectTimeout = (int)config.ConnectionTimeout.TotalMilliseconds;
```

Reply:
Not sure. I didn't intend to commit that file but it doesn't seem to do anything at the moment as I get disconnected all the time anyway.